### DISCUSSION TITLE
Conta 2FA GitHub

### DISCUSSION THREAD

**Discussion by filipedev6** [community/community#42665] [New to GitHub]
## Autenticação com 2FA GitHub

- Recentemente eu perdi meu numero antigo que estava sendo autenticado com meu GitHub e ontem fui formatar meu PC e perdi meu acesso ao GitHub também e não me lembrei de salvar 2FA que tinha no perfil e não tenho mais o txt que mandaram para mim quando habilitei, tentei de varias formas recuperar minha conta mais ate agora nenhum resultado já entrei em contato com GitHub pelo Email e ate agora nenhum resultado também já não sei mais oque faço seria muito triste eu perder tudo oque eu tinha na conta.

**Comment by SilasRodrigues19** [comment]
E ai @filipedev6 

Tenta usar essa opção do print abaixo, é pra redefinir suas configurações de autenticação e ai o GitHub vai analisar sua solicitação.

![image](https://user-images.githubusercontent.com/47532703/209565902-9e8fdb33-1c58-4604-9d1a-97472aca2861.png)

**Reply by filipedev6** [reply]
Já tentei utilizar esse e todos os métodos de recuperação mais nenhum foi possível ate agora
![image](https://user-images.githubusercontent.com/121410595/209566391-95fa3035-f295-448a-a9f1-8a6385d7b992.png)

**Comment by MullerHub** [comment]
Se eu não me engano você tinha 8 chaves de segurança de uso unico para essas situações na criação do 2FA, se salvou esses codigos em algum lugar, com eles você recupera sua conta com facilidade
